src/App.php

```diff
@@ -340,21 +334,20 @@ public function getBasePath(): string
 	 * @param L10n                        $l10n     The translator instance
 	 * @param App\Arguments               $args     The Friendica Arguments of the call
 	 * @param IManagePersonalConfigValues $pConfig  Personal configuration
-	 * @param IHandleSessions             $session  The Session handler
-	 */
-	public function __construct(Database $database, IManageConfigValues $config, App\Mode $mode, BaseURL $baseURL, LoggerInterface $logger, Profiler $profiler, L10n $l10n, Arguments $args, IManagePersonalConfigValues $pConfig, IHandleSessions $session, IHandleUserSessions $userSession)
-	{
-		$this->database    = $database;
-		$this->config      = $config;
-		$this->mode        = $mode;
-		$this->baseURL     = $baseURL;
-		$this->profiler    = $profiler;
-		$this->logger      = $logger;
-		$this->l10n        = $l10n;
-		$this->args        = $args;
-		$this->pConfig     = $pConfig;
-		$this->session     = $session;
-		$this->userSession = $userSession;
+	 * @param IHandleUserSessions         $sessions The (User)Session handler
+	 */
+	public function __construct(Database $database, IManageConfigValues $config, App\Mode $mode, BaseURL $baseURL, LoggerInterface $logger, Profiler $profiler, L10n $l10n, Arguments $args, IManagePersonalConfigValues $pConfig, IHandleUserSessions $sessions)
+	{
+		$this->database = $database;
+		$this->config   = $config;
+		$this->mode     = $mode;
+		$this->baseURL  = $baseURL;
+		$this->profiler = $profiler;
+		$this->logger   = $logger;
+		$this->l10n     = $l10n;
+		$this->args     = $args;
+		$this->pConfig  = $pConfig;
+		$this->session  = $sessions;
```

Comment:
I would keep the singular for the parameter name, the same as the attribute.
```suggestion
	public function __construct(Database $database, IManageConfigValues $config, App\Mode $mode, BaseURL $baseURL, LoggerInterface $logger, Profiler $profiler, L10n $l10n, Arguments $args, IManagePersonalConfigValues $pConfig, IHandleUserSessions $session)
	{
		$this->database = $database;
		$this->config   = $config;
		$this->mode     = $mode;
		$this->baseURL  = $baseURL;
		$this->profiler = $profiler;
		$this->logger   = $logger;
		$this->l10n     = $l10n;
		$this->args     = $args;
		$this->pConfig  = $pConfig;
		$this->session  = $session;
```